main.go

```diff
@@ -75,17 +74,18 @@ func main() {
 
 	// holds references for activated providers for current user token
 	scms := auth.NewScms()
-	bh := web.BaseHookOptions{
-		BaseURL: *baseURL,
-		Secret:  os.Getenv("WEBHOOK_SECRET"),
-	}
+	// bh := web.BaseHookOptions{
+	// 	BaseURL: *baseURL,
+	// 	Secret:  os.Getenv("WEBHOOK_SECRET"),
+	// }
 
 	runner, err := ci.NewDockerCI(logger)
 	if err != nil {
 		log.Fatalf("failed to set up docker client: %v\n", err)
 	}
 	defer runner.Close()
 
+	// TODO: find and replace (if possible) all occasions where this tiken is used
```

Comment:
This is used only by the `cmd/approvelist` command and the `helpbot` (separate repo). See `NewQuickFeed()` and `getSubmissions` in `approvelist`.

Reply:
There is also some commented code in `GetSubmissionsByCourse` that should be removed when this function can be used from `cmd/approvelist` without auth problems.